src/main/kotlin/nl/info/client/smartdocuments/SmartDocumentsClient.kt

```diff
@@ -34,6 +34,13 @@ interface SmartDocumentsClient {
         deposit: Deposit
     ): AttendedResponse
 
+    @POST
+    @Path("/wsxmldeposit/deposit/wizard_no_auth")
+    fun attendedDepositWizardNoAuth(
```

Comment:
Sorry, just realized. Very minor though. 
```suggestion
    fun attendedDepositNoAuth(
```

Because `attended` and `wizard` are two terms for the very same thing. Too bad SmartDocuments makes this so confusing..

Reply:
NP; fixed